lib/index.js

```diff
@@ -188,9 +180,9 @@ module.exports = class DockerComposeParser {
       },
       []
     )
-    DockerComposeParser._checkIfMissingMainToArray(services, opts)
     const envFiles = DockerComposeParser._getAllENVFiles(services)
-    return { results: services, envFiles }
+    const mains = DockerComposeParser._getMains(services, opts)
+    return { results: services, envFiles, mains }
```

Comment:
maybe we can rename `results` to `services` since we already changing interface in this PR. But that is not required change for this PR